### DISCUSSION TITLE
eccodes fail to build on Linux Mint 20 (ulyana)

### DISCUSSION THREAD

**Discussion by alexjcwitsil** [Homebrew/discussions#71] [Everyday usage] [answered]
Hello everyone in this wonderful community.  
I am new to brew and am attempting to install eccodes with 

`brew install eccodes`

However I get a 'Configuring incomplete' message.  I've copied and pasted that message below and also included a link to generated logs from `brew gist-logs`:

https://gist.github.com/alexjcwitsil/1c3e0b218de86bff2fb43f739471bc58

-- Configuring incomplete, errors occurred!
See also "/tmp/eccodes-20201017-16318-15tm1ez/eccodes-2.19.0-Source/build/CMakeFiles/CMakeOutput.log".
See also "/tmp/eccodes-20201017-16318-15tm1ez/eccodes-2.19.0-Source/build/CMakeFiles/CMakeError.log".

My version of brew is up to date: 
$ brew update
Already up-to-date.

And everything seems to be in order within brew: 
$ brew doctor
Your system is ready to brew.

As stated in the topic, I am running Linux Mint release 20 (ulyana) but if you need more specifics, I can certainly provide those.  Thanks everyone!

**Comment by jonchang** [comment, marked as answer]
Looks like this is related to a flag we added to fix the build on GCC 10 on macOS, but it doesn't work on Linux, which uses GCC 5. https://github.com/Homebrew/homebrew-core/commit/f4233e723edc28678ba8917f22f92d132c5961e9

Could you open a pull request to Homebrew/homebrew-core that removes the ENV.prepend lines? If it builds on macOS, then that change will be merged to the Linux repository as well, fixing the build for Linux users. [This document](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request) should help you get started. Thanks!

**Reply by alexjcwitsil** [reply]
It took several attempts, and I am sure I gave some maintainers a few headaches, but I think I was able to follow your instructions to 

1. edit Formula/eccodes.rb
2. submit a pull request to homebrew-core

I was able to build the formula locally without error so I guess we will see if it as accepted or not. 

Thanks for your help!

**Comment by chenrui333** [comment]
I am bumping eccodes to 2.19.1 now, https://github.com/Homebrew/homebrew-core/pull/64471/.

Let's see how it works out.
